src/vs/workbench/parts/terminal/electron-browser/terminalInstance.ts

```diff
@@ -568,6 +568,24 @@ export class TerminalInstance implements ITerminalInstance {
 		document.execCommand('paste');
 	}
 
+	public sendFile(filePath: string, addNewLine: boolean): void {
+		this._processReady.then(() => {
+			let path: string;
+			if (/(System32|Sysnative)\\(bash.exe|wsl.exe)/i.exec(this._shellLaunchConfig.executable)) {
+				path = `"${filePath.replace(/([A-Za-z]):/, (m, g) => `mnt/${g.toLocaleLowerCase()}`)}"`;
```

Comment:
`toLowerCase` is probably more correct than `toLocaleLowerCase` here since it's directory

Reply:
you got it!